reference/price_lists.v3.yml

```diff
@@ -2692,6 +2717,31 @@ paths:
         required: true
         schema:
           type: string
+  "/pricelists/{price_list_id}/assignments":
+    put:
+      tags:
+        - Price Lists Assignments
+      description: Upsert a single `Price List Assignment` for a `Price List`.
+      summary: Upsert Price List Assignment
+      operationId: upsertPriceListAssignment
+      parameters:
+        - $ref: "#/components/parameters/PriceListIdParam"
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/AssignmentForPutRequest"
+        required: true
+      responses:
+        "200":
+          description: A price list assignment.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/AssignmentForPutResponse"
```

Comment:
`PUT` `200` response appears to be missing the `data` and `meta` objects. This is the response I received:
```json
{
    "data": {
        "id": 11,
        "price_list_id": 4,
        "customer_group_id": 2,
        "channel_id": 1
    },
    "meta": {}
}
```
Within Stoplight, the response example is defaulting to negative integers. Can we either set a min or put in some example values? 
![image](https://user-images.githubusercontent.com/29524703/202566192-1b871d90-3679-4249-baee-56be49c56fa0.png)

Reply:
good catch, I updated the PUT 200 response and also added example values for successful PUT and GET responses